### ISSUE TITLE
Grid not big enough for uppercase accent mark letterforms

### ISSUE THREAD

**Issue by CrisA-Works** [javierbyte/brutalita#5]
There is enough room for creating lowercase á é í ó ú, but not enough for Á É Í Ó Ú

**Comment by javierbyte** [comment]
@alarcondgtuc agree, I'm working on a setting that would let you expand the grid, I think that it is going to be better than expanding the grid by one or two rows now.

**Comment by raphaelbastide** [comment]
I found that changing the grid size is quite easy by [tweaking the source code](https://github.com/javierbyte/brutalita/blob/master/src/app.jsx#L47) a little. What I have more difficulties to find, is a function that can batch move all my existing glyphs in X and Y. A little help would be very welcome.

**Comment by neokeld** [comment]
See here now: https://github.com/javierbyte/brutalita/blob/a9aef0e19214dc7313a0d20b403a20c932c80987/src/app.tsx#L116
